brickflow/codegen/databricks_bundle.py

```diff
@@ -505,12 +548,55 @@ def _build_native_spark_jar_task(
             **task.cluster.job_task_field_dict,
         )
 
+    def _build_native_spark_python_task(
+        self,
+        task_name: str,
+        task: Task,
+        task_libraries: List[JobsTasksLibraries],
+        task_settings: TaskSettings,
+        depends_on: List[JobsTasksDependsOn],
+        **kwargs: Any,
+    ) -> JobsTasks:
+        try:
+            spark_python_task: JobsTasksSparkPythonTask = task.task_func()
+        except Exception as e:
+            raise ValueError(
```

Comment:
Can you please add tests for this as well

Reply:
Test has been added